src/st/mad/11D3C.c

```diff
@@ -303,35 +303,7 @@ void EntityExplosionSpawn(u16 arg0, u16 arg1) {
     g_CurrentEntity->step_s = 0;
 }
 
-void InitializeEntity(u16 arg0[]) {
-    u16 enemyId;
-    EnemyDef* enemyDef;
-
-    g_CurrentEntity->animSet = *arg0++;
-    g_CurrentEntity->animCurFrame = *arg0++;
-    g_CurrentEntity->unk5A = *arg0++;
-    g_CurrentEntity->palette = *arg0++;
-
-    enemyId = *arg0++;
-    g_CurrentEntity->enemyId = enemyId;
-    enemyDef = &g_api.enemyDefs[enemyId];
-    g_CurrentEntity->hitPoints = enemyDef->hitPoints;
-    g_CurrentEntity->attack = enemyDef->attack;
-    g_CurrentEntity->attackElement = enemyDef->attackElement;
-    g_CurrentEntity->hitboxState = enemyDef->hitboxState;
-    g_CurrentEntity->hitboxWidth = enemyDef->hitboxWidth;
-    g_CurrentEntity->hitboxHeight = enemyDef->hitboxHeight;
-    g_CurrentEntity->flags = enemyDef->flags;
-    g_CurrentEntity->hitboxOffX = 0;
-    g_CurrentEntity->hitboxOffY = 0;
-    g_CurrentEntity->step_s = 0;
-    g_CurrentEntity->step++;
-    if (g_CurrentEntity->zPriority == 0) {
-        g_CurrentEntity->zPriority =
-            g_unkGraphicsStruct.g_zEntityCenter.unk - 0xC;
-    }
-}
-
+#include "../init_entity.h"
```

Comment:
Nice job catching these other versions! Could you also add `rwrp`, in `src\st\rwrp\D580.c`, line 236?

Reply:
Could also do `src/st/wrp/st_common.h`.

Reply:
Good catch on the `INCLUDE_ASM` in `rwrp`. Fixed that.